src/System.IO.FileSystem/tests/File/Delete.cs

```diff
@@ -70,6 +70,7 @@ public void PositiveTest()
         [Fact]
         public void NonExistentFile()
         {
+            Delete(Path.Combine(Path.GetPathRoot(TestDirectory), Path.GetRandomFileName()));
```

Comment:
Why GetPathRoot?

Reply:
My idea is to add test similar to https://github.com/dotnet/corefx/pull/32660/files#diff-0fe3419b22b4609b424cd8730bf45893R108
Case with "no directory"...do you think it's unuseful?